lib/scala/project-manager/src/test/scala/org/enso/projectmanager/ProjectManagementOps.scala

```diff
@@ -79,20 +79,28 @@ trait ProjectManagementOps { this: BaseServerSpec =>
   def openProjectData(implicit client: WsTestClient): ProjectOpen.Result = {
     val Right(openReply) = parse(client.expectMessage(20.seconds.dilated))
     val openResult = for {
-      result    <- openReply.hcursor.downExpectedField("result")
-      engineVer <- result.downField("engineVersion").as[SemVer]
-      jsonAddr  <- result.downExpectedField("languageServerJsonAddress")
-      jsonHost  <- jsonAddr.downField("host").as[String]
-      jsonPort  <- jsonAddr.downField("port").as[Int]
-      binAddr   <- result.downExpectedField("languageServerBinaryAddress")
-      binHost   <- binAddr.downField("host").as[String]
-      binPort   <- binAddr.downField("port").as[Int]
-      name      <- result.downField("projectName").as[String]
-      namespace <- result.downField("projectNamespace").as[String]
+      result        <- openReply.hcursor.downExpectedField("result")
+      engineVer     <- result.downField("engineVersion").as[SemVer]
+      jsonAddr      <- result.downExpectedField("languageServerJsonAddress")
+      jsonHost      <- jsonAddr.downField("host").as[String]
+      jsonPort      <- jsonAddr.downField("port").as[Int]
+      binAddr       <- result.downExpectedField("languageServerBinaryAddress")
+      binHost       <- binAddr.downField("host").as[String]
+      binPort       <- binAddr.downField("port").as[Int]
+      projectName   <- result.downField("projectName").as[String]
+      projectModule <- result.downField("projectModule").as[String]
```

Comment:
Is this changing content of `package.yaml` file? If so, then update `/docs/distribution/packaging.md` as that seems to contain the specification of the content of that file.